changelog/README.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 30th January 2025
+* [Add rates](../operations/rates.md#add-rates) - Introduced backward-compatible [Hybrid identifier](../operations/_objects.md#hybrid-identifier) string for request parameters `ServiceId`, `RateGroupId` and `BaseRateId`.
+* [Set rates](../operations/rates.md#set-rates) - Introduced backward-compatible [Hybrid identifier](../operations/_objects.md#hybrid-identifier) string for request parameters `ServiceId`, `RateGroupId` and `BaseRateId`.
+* [Delete rates](../operations/rates.md#delete-rates) - Introduced backward-compatible [Hybrid identifier](../operations/_objects.md#hybrid-identifier) string for request parameter `RateIds`.
+* [Get all age categories](../operations/agecategories.md#get-all-age-categories) - Extended [Age category](../operations/agecategories.md#age-category) with `ExternalIdentifier`.

```

Comment:
Adjust this to a new format, see few last entries in the changelog, it should be basically
```
* operation1
  * change1
  * change2
* operation2
  * change1
  * change2
```

Reply:
Hm, I saw this, but the [style guide](https://mews.atlassian.net/wiki/spaces/AD/pages/224628207/Style+Guide+for+API+Changelog) doesn't show it like this. I guess it is outdated?

Reply:
Yeah, definitely outdated, this change to format is recent and I guess @MikeAdamsMews did not updated that style guide 👀

Reply:
Updated.